docs/running-a-node/overview.md

```diff
@@ -14,27 +14,40 @@ Regardless of the motivation for running a node, it will only benefit the networ
 
 ## Node types
 
-### Principal Representative Nodes
-Currently, nodes configured with Representative accounts with at least 0.1% of the [online voting weight](/glossary#online-voting-weight) delegated to them participate more broadly in network consensus because they send votes to their peers which are subsequently rebroadcast.
+--8<-- "dedicated-representative-nodes.md"
+
+### Non-voting nodes
+When first setting up a node it will not be configured to participate in consensus by voting on traffic. This type of node is common and is recommended for all integrations. If your goal in setting up a node is to learn how to integrate and use Nano for payments, this is the best starting point. If you want to dedicate resources to help secure consensus on the network, then a Representative node should be explored.
+
+### Representative nodes
+If a node is setup with a Representative account, is configured to vote and has **less than 0.1% of [online voting weight](/glossary#online-voting-weight)** delegated to them, they are a considered Representative node. These nodes will validate and vote on transactions seen on the network; however, other nodes on the network will not rebroadcast their votes.
+
+### Principal Representative nodes
+Representative nodes with **at least 0.1% of the [online voting weight](/glossary#online-voting-weight)** delegated to them participate more broadly in network consensus because they send votes to their peers which are subsequently rebroadcast. These nodes have the most impact to the security and availability of the network so [keeping them secure](security.md) and following [maintenance recommendations](#maintenance) should be taken seriously.
 
 !!! success "Becoming a Principal Representative"
 	With the ability for any user on the network to redelegate their voting weight, even an account with no weight today can become a Principal Representative over time.
 
-### Representative Nodes
-Nodes with less than 0.1% of [online voting weight](/glossary#online-voting-weight) will validate and vote on transactions seen on the network; however, other peers on the network will not rebroadcast their votes.
+!!! tip "Review Node security guide"
+	Regardless of the type of node you are planning to run, make sure to review the [Node security guide](security.md) to ensure best practice with configuration, firewalls and more.
 
-## Resources and ongoing maintenance
+## Hardware recommendations
+<span id="resources-and-ongoing-maintenance"></span>
```

Comment:
Possibly leftover from testing something? Removed this and didn't make a difference in the output.

Reply:
The `id` in this span tag actually allows the older anchor `#resources-and-ongoing-maintenance` on a URL to continue to function while allowing the changing of the section title. I thought this might have been shared externally at some point so I put this in there to preserve it just in case.